### PR TITLE
Remove SwiftyJSON

### PR DESCRIPTION
**Pull request by ShihabMehboob**
This aims to replace SwiftyJSON with Codable.

### CODE REVIEW

**Comment by ShihabMehboob** [comment]
Do not merge yet

**Comment by sjanuary** [comment]
@ShihabMehboob could you rebase this PR please?  You seem to have some of Jonathan's changes in the diff that have already been merged into master.

Has this been tested on Bluemix to check that the autoscaling agent is still getting the right data from us?

**Comment by ShihabMehboob** [comment]
screen shot 2018-07-25 at 12 23 55

The app has been tested on Bluemix/IBM Cloud, and attached to the autoscaling service, displaying data as seen in the screenshot of the dashboard data.

File: AUTHORS.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
- - Ian Partridge (https://github.com/ianpartridge) 
+ - Ian Partridge (https://github.com/ianpartridge)
  - Toby Corbin (https://github.com/tobespc)
  - Julie Stalley (https://github.com/stalleyj)
  - Navneet Gupta (https://github.com/na-gupta)
  - Ricardo Olivieri (https://github.com/rolivieri)
  - qibobo (https://github.com/qibobo)  
  - Sian January (https://github.com/sjanuary)
  - Kye Maloy (https://github.com/kyemaloy97)
+ - Jonathan Spruce (https://github.com/jonathan-spruce)
```
Comment (ianpartridge) [inline AUTHORS.md]: Shouldn't this be adding your name, not Jonathan's?

File: Sources/SwiftMetricsBluemix/SwiftMetricsBluemix.swift [hunk 2]
```diff
@@ -443,21 +442,34 @@ public class SwiftMetricsBluemix {
...
   }
+    
+}
 
+public struct ConfigResponse: Codable {
+    var agent: [String]
+    var metricsConfig: String
+    var reportInterval: Int
+    
+    public enum CodingKeys: String, CodingKey {
```
Comment (ianpartridge) [inline Sources/SwiftMetricsBluemix/SwiftMetricsBluemix.swift]: As these CodingKeys don't change the name of any of the keys, can we remove the enum and let the compiler synthesise it?

**Comment by sjanuary** [comment]
@ShihabMehboob have you re-tested on Bluemix after making the changes that Ian asked for?  If so I'm happy to get this merged in.

**Comment by ShihabMehboob** [comment]
screen shot 2018-08-16 at 10 29 12

**Review by sjanuary — APPROVED** [review]
Now approved

**Pull request merged**